lib/flatpak-installation.c

```diff
@@ -225,7 +225,7 @@ flatpak_get_system_installations (GCancellable *cancellable,
       else
         {
           /* Warn about the problem and continue without listing this installation. */
-          g_warning ("Unable to create FlatpakInstallation for: %s", local_error->message);
+          g_warning ("Unable to create FlatpakInstallation for system: %s", local_error->message);
```

Comment:
Would it make sense to add the `flatpak_dir_get_name` output here?

Reply:
Yeah. Done.